packages/airnode-validator/src/v2/config/config.ts

```diff
@@ -44,7 +44,7 @@ export const priorityFeeSchema = z.object({
 
 export const chainOptionsSchema = z.object({
   txType: z.union([z.literal('legacy'), z.literal('eip1559')]),
-  baseFeeMultiplier: z.string().optional(),
+  baseFeeMultiplier: z.number().int().optional(),
```

Comment:
Zod also has more options for numbers, see [here](https://github.com/colinhacks/zod#numbers) (e.g. only positive numbers).